dom-creation/main.js

```diff
@@ -0,0 +1,95 @@
+    imageUrl: 'images/bulbasaur.png'
+  },
+  {
+    number: '004',
+    name: 'Charmander',
+    description: 'It has a preference for hot things. When it rains, steam is said to spout from the tip of its tail.',
+    imageUrl: 'images/charmander.png'
+  },
+  {
+    number: '007',
+    name: 'Squirtle',
+    description: 'When it retracts its long neck into its shell, it squirts out water with vigorous force.',
+    imageUrl: 'images/squirtle.png'
+  },
+  {
+    number: '002',
+    name: 'Ivysaur',
+    description: 'When the bulb on its back grows large, it appears to lose the ability to stand on its hind legs.',
+    imageUrl: 'images/ivysaur.png'
+  },
+  {
+    number: '005',
+    name: 'Charmeleon',
+    description: 'It has a barbaric nature. In battle, it whips its fiery tail around and slashes away with sharp claws.',
+    imageUrl: 'images/charmeleon.png'
+  },
+  {
+    number: '008',
+    name: 'Wartortle',
+    description: 'It is recognized as a symbol of longevity. If its shell has algae on it, that Wartortle is very old.',
+    imageUrl: 'images/wartortle.png'
+  },
+  {
+    number: '003',
+    name: 'Venusaur',
+    description: 'Its plant blooms when it is absorbing solar energy. It stays on the move to seek sunlight.',
+    imageUrl: 'images/venusaur.png'
+  },
+  {
+    number: '006',
+    name: 'Charizard',
+    description: 'It spits fire that is hot enough to melt boulders. It may cause forest fires by blowing flames.',
+    imageUrl: 'images/charizard.png'
+  },
+  {
+    number: '009',
+    name: 'Blastoise',
+    description: 'It crushes its foe under its heavy body to cause fainting. In a pinch, it will withdraw inside its shell.',
+    imageUrl: 'images/blastoise.png'
+  }
+];
+
+function renderPokemon(pokemon) {
+  var $newDivOne = document.createElement('div');
+  $newDivOne.setAttribute('class', 'column-third');
+
+  var $newDivTwo = document.createElement('div');
+  $newDivTwo.setAttribute('class', 'pokemon-card');
+  $newDivOne.appendChild($newDivTwo);
+
+  var $newImg = document.createElement('img');
+  $newImg.setAttribute('src', pokemon.imageUrl);
+  $newDivTwo.appendChild($newImg);
+
+  var $newDivThree = document.createElement('div');
+  $newDivThree.setAttribute('class', 'pokemon-card-text');
+  $newDivTwo.appendChild($newDivThree);
+
+  var $newHTwo = document.createElement('h2');
+  var $newTextOne = document.createTextNode(pokemon.name);
+  $newHTwo.appendChild($newTextOne);
+  $newDivThree.appendChild($newHTwo);
+
+  var $newHThree = document.createElement('h3');
+  var $newTextTwo = document.createTextNode('#' + pokemon.number);
+  $newHThree.appendChild($newTextTwo);
+  $newDivThree.appendChild($newHThree);
+
+  var $newP = document.createElement('p');
+  var $newTextP = document.createTextNode(pokemon.description);
+  $newP.appendChild($newTextP);
+  $newDivThree.appendChild($newP);
+  return $row.appendChild($newDivOne);
```

Comment:
You should not be appending to `$row` inside this function. You should only be returning the `column-third` div. Make sure you read all of the instructions correctly